src/lsm/grid.zig

```diff
@@ -144,17 +145,19 @@ pub fn GridType(comptime Storage: type) type {
         cache: Cache,
 
         write_iops: IOPS(WriteIOP, write_iops_max) = .{},
-        write_queue: FIFO(Write) = .{},
+        write_iop_tracer_slots: [write_iops_max]?tracer.SpanStart = .{null} ** write_iops_max,
+        write_queue: FIFO(Write) = .{ .name = "grid_write" },
 
         // Each read_iops has a corresponding block.
         read_iop_blocks: [read_iops_max]BlockPtr,
         read_iops: IOPS(ReadIOP, read_iops_max) = .{},
-        read_queue: FIFO(Read) = .{},
+        read_iop_tracer_slots: [read_iops_max]?tracer.SpanStart = .{null} ** read_iops_max,
```

Comment:
Nice, I didn't know that this syntax worked for initializing arrays — I've always done `[_]T{value} ** count`, but `.{value} ** count` is much nicer.